src/discof/restore/utils/fd_ssmsg.h

```diff
@@ -212,7 +212,7 @@ struct fd_snapshot_manifest {
      Currently, these parameters can never change and are fixed from
      genesis onwards, although in future they may change with new
      feature flags. */
-  fd_snapshot_manifest_epoch_schedule_params_t epoch_schedule_params;
+  fd_epoch_schedule_t epoch_schedule_params;
```

Comment:
i would not change the ssmsg schema here (believe snapshot team intentionally made a separate type) and instead do a type pun cast at the usage site in publish_stake_weights